nengo_viz/static/viz_modal.js

```diff
@@ -22,7 +22,13 @@ VIZ.Modal.footer = function(type){
 
     if (type === "close") {
         $footer.append('<button type="button" class="btn btn-default" data-dismiss="modal">Close</button>');
-    } else {
+    } 
+    else if (type === 'confirm_reset') {
+        $footer.append('<button type="button" id="confirm_reset_button" class="btn btn-primary">Reset</button>');
+        $footer.append('<button type="button" class="btn btn-default" data-dismiss="modal">Close</button>');
+        $('#confirm_reset_button').on('click', function() {toolbar.reset_model_layout();});
```

Comment:
Oops, I just merged this.  Damnit.  I completely agree with your comment but I figured people were waiting till a future PR to fix that up.  In any case, if it gets fixed here then I can cherry-pick the change into master too.